doc/man3/SSL_CIPHER_get_name.pod

```diff
@@ -88,6 +92,18 @@ TLS 1.3 cipher suites) B<NID_auth_any> is returned. Examples (not comprehensive)
 SSL_CIPHER_is_aead() returns 1 if the cipher B<c> is AEAD (e.g. GCM or
 ChaCha20/Poly1305), and 0 if it is not AEAD.
 
+SSL_CIPHER_find() returns a B<SSL_CIPHER> structure which has the cipher ID stored in
+B<ptr>. The B<ptr> parameter is a two elements array of B<char>, which stores the two
+bytes TLS cipher ID in network byte order. This parameter is usually retrieved from a
+TLS packet by using functions like L<PACKET_get_bytes(3)>. SSL_CIPHER_find() returns
```

Comment:
PACKET_get_bytes() is not a public function and is highly unlikely to become one; I don't think we should mention it here.
SSL_early_get0_ciphers() would be a better example.